<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Courier_Main</class>
 <widget class="QMainWindow" name="Courier_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>我的任务</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QFrame" name="map_frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>500</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/map_bg.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>491</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="package_id">
       <property name="placeholderText">
        <string>包裹id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="time">
       <property name="placeholderText">
        <string>完成时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submit">
       <property name="text">
        <string>完成</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="weather">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>天气</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_my_asks">
   <property name="text">
    <string>my_asks</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
